nsv13/code/modules/overmap/knpc.dm

```diff
@@ -383,6 +400,9 @@ This is to account for sec Ju-Jitsuing boarding commandos.
 
 /datum/ai_goal/human/proc/reload(datum/component/knpc/HA, obj/item/gun)
 	var/mob/living/carbon/human/ai_boarder/H = HA.parent
+	if(world.time < H.next_attack)
```

Comment:
This code is far too static and does not consider the dynamic scaling code of KNPC spawns.

Reply:
Thanks for the feedback. Changed it to be handled the same way that movement speed is in the current version.